integration/client-cli/admin-cli/src/main/java/org/keycloak/client/admin/cli/commands/ConfigCmd.java

```diff
@@ -16,66 +16,35 @@
  */
 package org.keycloak.client.admin.cli.commands;
 
-import org.jboss.aesh.cl.GroupCommandDefinition;
-import org.jboss.aesh.console.command.CommandException;
-import org.jboss.aesh.console.command.CommandResult;
-import org.jboss.aesh.console.command.invocation.CommandInvocation;
-
 import java.io.PrintWriter;
 import java.io.StringWriter;
 
+import picocli.CommandLine.Command;
+
 import static org.keycloak.client.admin.cli.util.OsUtil.CMD;
-import static org.keycloak.client.admin.cli.util.OsUtil.EOL;
 
 
 /**
  * @author <a href="mailto:mstrukel@redhat.com">Marko Strukelj</a>
  */
 
-@GroupCommandDefinition(name = "config", description = "COMMAND [ARGUMENTS]", groupCommands = {ConfigCredentialsCmd.class} )
+@Command(name = "config", description = "COMMAND [ARGUMENTS]", subcommands = {
+        ConfigCredentialsCmd.class,
+        ConfigTruststoreCmd.class
+} )
 public class ConfigCmd extends AbstractAuthOptionsCmd {
 
-    public CommandResult execute(CommandInvocation commandInvocation) throws CommandException, InterruptedException {
-        try {
-            if (args != null && args.size() > 0) {
-                String cmd = args.get(0);
-                switch (cmd) {
-                    case "credentials": {
-                        args.remove(0);
-                        ConfigCredentialsCmd command = new ConfigCredentialsCmd();
-                        command.initFromParent(this);
-                        return command.execute(commandInvocation);
-                    }
-                    case "truststore": {
-                        args.remove(0);
-                        ConfigTruststoreCmd command = new ConfigTruststoreCmd();
-                        command.initFromParent(this);
-                        return command.execute(commandInvocation);
-                    }
-                    default: {
-                        if (printHelp()) {
-                            return help ? CommandResult.SUCCESS : CommandResult.FAILURE;
-                        }
-                        throw new IllegalArgumentException("Unknown sub-command: " + cmd + suggestHelp());
```

Comment:
The current error message is not as pretty as the old one:

Execute command with a typo:
```
./kcadm.sh config truststores
```

Error message:
```
Unmatched argument at index 1: 'truststores'
```

Reply:
I'd argue that it doesn't seem worth introducing additional parsing logic for this case since it will be clear from the help what is needed. I think this also may be a case where the default picocli error is probably better than the short one we are producing.

Reply:
Ok, works for me.